### PR TITLE
newfile created

### PR DESCRIPTION
**Pull request by lfundaro**

### CODE REVIEW

File: newFile.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (lfundarotest) [inline newFile.json:1]: can we make a change here ?

**Review by lfundarotest — CHANGES_REQUESTED** [review]
what do you think about making that change ?

File: newFile.json (hunk 1 above)
Comment (lfundaro) [inline newFile.json:1]: there is something here we could add.

**Review by lfundaro — COMMENTED** [review]
this is a comment, not a review

File: newFile.json (hunk 1 above)
Comment (lfundarotest) [inline newFile.json:1]: I would like to rephrase this part.

**Review by lfundarotest — CHANGES_REQUESTED** [review]
this is a review summary.

File: newFile.json (hunk 1 above)
Comment (lfundarotest) [inline newFile.json:1]: I have yet another comment to make here

**Review by lfundarotest — CHANGES_REQUESTED** [review]
this is the summary of my PR

**Review by lfundarotest — APPROVED** [review]

**Pull request merged**